### ISSUE TITLE
Add JMAP support

### ISSUE THREAD

**Issue by mildred** [Mailu/Mailu#471]
[JMAP](https://jmap.io/) would be great to support. Seems [dovecot will include it in v2.3](https://old.lwn.net/Articles/687159/)

**Comment by kaiyou** [comment]
I am waiting on JMAP indeed, I really hope this will improve the current situation with email clients. I also hope that webmails will support accessing JMAP directly as a subpath.

**Comment by xamanu** [comment]
[K9 is going to have added jmap support] (https://prototypefund.de/project/jmap-for-k9-mail/) within next 1/2 a year. It would be nice to have my beloved email server being able to offer the corresponding piece to save energy on the smartphone.

Thanks for this great piece of software!

**Comment by DakshMiglani** [comment]
@kaiyou any updates on this?

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by Diman0** [comment]
@DakshMiglani @xamanu 
Mailu makes use of dovecot as IMAP server. To get support for JMAP in Mailu it must be implemented in dovecot first. Feel free to contact the developers of dovecot to request support for JMAP: https://www.dovecot.org/mailing-lists.

**Comment by xamanu** [comment]
Yeah, cool. Thanks for looking into that. For me it is fine to close. Anyway it is just a nice to have.

**Comment by Litengat** [comment]
4 years later any updates on this ?

**Comment by alexanderadam** [comment]
> 4 years later any updates on this ?

Well, kind of: there's [a performant and secure FOSS mail server called Stalwart](https://github.com/stalwartlabs/mail-server?tab=readme-ov-file#features) which supports JMAP.

**Comment by nextgens** [comment]
We now ship dovecot 2.3... unfortunately JMAP support is still not implemented upstream.

Tell upstream (dovecot) why you need it.

**Comment by Litengat** [comment]
imap sucks to implement with js and jmap is mutch better
